assets/js/base/components/radio-control/types.ts

```diff
@@ -24,7 +24,7 @@ export interface RadioControlOptionProps {
 }
 
 interface RadioControlOptionContent {
-	label: string;
+	label: string | JSX.Element;
```

Comment:
I don't think we should be allowing a React element to be used as a label here, only strings. We can modify the `RadioControlAccordionProps` so that `option.label` only takes a string

Reply:
Ah you are right, it looks like we do allow label to be a component as well. In that case, let's leave as it is!